src/scrape_up/github/issue.py

```diff
@@ -67,4 +66,10 @@ def title(self):
         except:
             message = "No title found"
             return message
-
+        
+    def opened_at(self):
+        """
+        Returns a string containing the time when the issue was opened in ISO format
+        """
+        data = self.__scrape_page()
+        return data.find('relative-time').text
```

Comment:
Add the `try-exception` method to handle any unwanted error.

Reply:
updated